README.md

```diff
@@ -467,3 +468,39 @@ They are as follows:
 [flipcoin code example]: https://github.com/opendatahub-io/data-science-pipelines-operator/blob/main/docs/example_pipelines/condition.py
 [installodh]: https://opendatahub.io/docs/quick-installation
 [kfp-tekton]: https://github.com/kubeflow/kfp-tekton
+
+# Deployment and Testing Guidelines for Developers
+
+To run the DSPO locally :
+
+Clone the directory and execute the following command:
+
+```bash
+go run main.go --config=$config.yaml
+```
+Below is the sample config file(the tags for the images can be edited as required):
+```bash
+Images:
+  ApiServer: quay.io/opendatahub/ds-pipelines-api-server:main-0e8a011
```

Comment:
we should update these images to use latest tags (rather than explicit SHAs/dev tags) where applicable